.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+### All Submissions:
+
+* [ ] Have you followed the guidelines in our [Contributing document](CONTRIBUTING.md)?
+* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
+* [ ] Does your PR link to an issue?
+
+<!-- Erase any parts of this template not applicable to your Pull Request. -->
+
+### New Notebook Submissions:
+
+* [ ] Have you included a summary of the notebook in the README.md included updated links to the notebook?
+* [ ] Does your PR include links to the new notebook **(in the branch)** for review using [nbviewer](https://nbviewer.jupyter.org/), [Colab](https://colab.research.google.com/), and [reviewnb](https://www.reviewnb.com/)? These three are the preferred ways to review changes and additions to notebooks during review.
+
+### Bug Fix Submissions:
+
+* [ ] Does your PR include links to the updated notebook **(in the branch)** for review using [nbviewer](https://nbviewer.jupyter.org/), [Colab](https://colab.research.google.com/), and [reviewnb](https://www.reviewnb.com/)? These three are the preferred ways to review changes and additions to notebooks during review.
```

Comment:
Maybe we can also add this part (or similarily) to the `Contributing.md` file (see comment above)